go.mod

```diff
@@ -0,0 +1,58 @@
+module knative.dev/net-certmanager
+
+go 1.14
```

Comment:
Please keep it as `go 1.13`, we haven't started switching CI to use `go 1.14` yet.

Reply:
Done